components/AssetFormBuy/AssetFormBuy.styled.ts

```diff
@@ -0,0 +1,65 @@
+import styled from 'styled-components';
+
+type DropdownMenuProps = {
+  isLoading: boolean;
+};
+
+export const General = styled.p`
+  color: #7578b5;
+  font-size: 14px;
+  line-height: 24px;
+  font-family: GilroyMedium;
```

Comment:
Can we move all of these font-families into global so that by default we're using font-family GilroyMedium?